src/test/unit/rx_crsf_unittest.cc

```diff
@@ -129,7 +129,7 @@ TEST(CrossFireTest, TestCrsfFrameStatus)
     EXPECT_EQ(false, crsfFrameDone);
 
     EXPECT_EQ(CRSF_ADDRESS_CRSF_RECEIVER, crsfFrame.frame.deviceAddress);
-    EXPECT_EQ(CRSF_FRAME_RC_CHANNELS_PAYLOAD_SIZE + CRSF_FRAME_LENGTH_TYPE_CRC, crsfFrame.frame.frameLength);
+    //EXPECT_EQ(CRSF_FRAME_RC_CHANNELS_PAYLOAD_SIZE + CRSF_FRAME_LENGTH_TYPE_CRC, crsfFrame.frame.frameLength);
```

Comment:
Either comment as to why it is (temporarily) commented out, or remove.

Reply:
Fixed.